java/client/src/org/openqa/selenium/support/ui/FluentWait.java

```diff
@@ -68,14 +69,18 @@
  */
 public class FluentWait<T> implements Wait<T> {
 
-  public static final Duration FIVE_HUNDRED_MILLIS = new Duration(500, MILLISECONDS);
+  public final static long DEFAULT_SLEEP_TIMEOUT = 500;
+
+  public static final Duration FIVE_HUNDRED_MILLIS = new Duration(DEFAULT_SLEEP_TIMEOUT, MILLISECONDS);
+
+  public static final java.time.Duration DEFAULT_WAIT_DURATION = java.time.Duration.ofMillis(DEFAULT_SLEEP_TIMEOUT);
```

Comment:
And so we don't need this to be `public`.

Reply:
it's used in `org.openqa.selenium.lift.WebDriverTestContext`, do you propose to create another constant in the mentioned class?